src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -62,21 +61,15 @@ class TestChannel(AioTestBase):
             self.assertIsInstance(response, messages_pb2.SimpleResponse)
 
     async def test_unary_call_times_out(self):
```

Comment:
nit: We could have a time out test case for succeed RPC just like before. You could refer to `server_test.py`. Not a blocker for this PR.

Reply:
The only "problem" is that our test suite for the client is quite coupled to this basic to the `test_pb2_grpc.py` file, so any change in functionality for any method could change the original semantic, the same that happened with the `EmptyCall` method.

This is a recurrent question, should we decouple the test suite from that file, so no longer use it, and use our own one? or even better, use our one server fixture based on a programmatic RPC handlers builder,  the same that is being used for testing the aio server currently.